
 This patch fixes bug #1031 about wrong pool source information
 in job report.

 Apply the patch to 2.2.8 (and possibly any 2.2.x version with):

 cd <bacula-source>
 patch -p0 <2.2.8-pool-source.patch
 ./configure <your-options>
 make
 ...
 make install



Index: src/dird/ua_run.c
===================================================================
--- src/dird/ua_run.c	(révision 6388)
+++ src/dird/ua_run.c	(copie de travail)
@@ -122,6 +122,9 @@
    jcr->verify_job = rc.verify_job;
    jcr->previous_job = rc.previous_job;
    jcr->pool = rc.pool;
+   if (jcr->pool != jcr->job->pool) {
+      pm_strcpy(jcr->pool_source, _("User input"));
+   }
    set_rwstorage(jcr, rc.store);
    jcr->client = rc.client;
    pm_strcpy(jcr->client_name, rc.client->name());
